packages/graphql-server/src/plugins/useRedwoodRealtime.ts

```diff
@@ -0,0 +1,61 @@
+import type { Plugin } from '@envelop/core'
+import type { UseLiveQueryOptions } from '@envelop/live-query'
+import { useLiveQuery } from '@envelop/live-query'
+import { mergeSchemas } from '@graphql-tools/schema'
+import { astFromDirective } from '@graphql-tools/utils'
+import type { PubSub } from '@graphql-yoga/subscription'
+import { createPubSub } from '@graphql-yoga/subscription'
+import { GraphQLLiveDirective } from '@n1ru4l/graphql-live-query'
+import { InMemoryLiveQueryStore } from '@n1ru4l/in-memory-live-query-store'
+import { print } from 'graphql'
+
+import type { SubscriptionGlobImports } from 'src/subscriptions/makeSubscriptions'
+
+export type { PubSub }
+
+export { createPubSub, InMemoryLiveQueryStore }
+
+export const liveDirectiveTypeDefs = print(
+  astFromDirective(GraphQLLiveDirective)
+)
+
+export type RedwoodRealtimeOptions = {
+  liveQueries?: UseLiveQueryOptions
+  /**
+   * @description Subscriptions passed from the glob import:
+   * import subscriptions from 'src/subscriptions/**\/*.{js,ts}'
+   */
+  subscriptions?: {
+    subscriptions: SubscriptionGlobImports
+    pubSub: PubSub<NonNullable<unknown>>
```

Comment:
FYI This may not be the proper type. Using for the moment, but ideally it would be


```export type PubSubChannels = {
  newMessage: [payload: { from: string; body: string }]
}

export const pubSub = createPubSub<PubSubChannels>()
```

the type of the createPubSub you gave it,

Reply:
Perhaps `    pubSub: ReturnType<typeof createPubSub>` is better? Trying.